src/main/native/cpp/support/Base64.cpp

```diff
@@ -62,42 +62,37 @@
  *
  */
 
-#include "support/Base64.h"
-
-#include "llvm/SmallVector.h"
-#include "llvm/raw_ostream.h"
+#include <llvm/SmallVector.h>
```

Comment:
Should not use <>, this is part of the same library as the llvm code.

Reply:
Fixed.